quantum/keycode_config.h

```diff
@@ -30,6 +30,7 @@ typedef union {
     struct {
         bool swap_control_capslock:1;
         bool capslock_to_control:1;
+        bool swap_control_lgui:1;
```

Comment:
Also, by placing this in the middle of this struct, it will cause users with Left Alt and Left GUI swapped to have their Left Control swapped with Left GUI instead, as everything will be shifted over by one bit. We can either wait for a breaking change release to clean up the ordering of this struct, or place it after the NKRO bit.

Reply:
My bad, I can move it to the end